timelock-server/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosTimeLockServer.java

```diff
@@ -112,16 +112,17 @@ private void registerLeaderElectionService(TimeLockServerConfiguration configura
 
         Set<String> paxosSubresourceUris = PaxosTimeLockUriUtils.getLeaderPaxosUris(remoteServers);
 
-        Leaders.LocalPaxosServices localPaxosServices = Leaders.createLocalServices(
+        Leaders.LocalPaxosServices localPaxosServices = Leaders.createInstrumentedLocalServices(
                 leaderConfig,
                 ImmutableRemotePaxosServerSpec.builder()
                         .remoteLeaderUris(remoteServers)
                         .remoteAcceptorUris(paxosSubresourceUris)
                         .remoteLearnerUris(paxosSubresourceUris)
-                        .build());
+                        .build(),
+                "paxos-leader-election-service");
```

Comment:
so this looks like it tries to use the package name - i think we actually want to distinguish between the leader election service and the timestamp bound store